frontend/src/components/ToolTip/ToolTip.style.ts

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+
+export const ToolTipText = styled.span<{ color: string; bgcolor: string; invertPosition: boolean }>`
+  background-color: ${props => props.color};
+  color: ${props => props.bgcolor};
+  visibility: hidden;
+  width: 120px;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+  top: ${props => (props.invertPosition ? '130%' : '-50%')};
+  left: 50%;
+  margin-left: -60px;
+  &:after {
+    content: '';
+    position: absolute;
+    bottom: ${props => (props.invertPosition ? '100%' : '-30%')};
+    left: 50%;
+    margin-left: -5px;
+    border-width: 5px;
+    border-style: solid;
+    border-color: ${props =>
+      props.invertPosition
+        ? `transparent transparent ${props.color} transparent`
+        : `${props.color} transparent transparent transparent`};
+
+`;
+
+export const ToolTip = styled('div')({
+  position: 'relative',
+  display: 'inline-block',
+  ':hover .tooltipSpan': {
```

Comment:
Thanks for the update!
I think we can make it even more robust. 
```suggestion
  ':hover > .tooltipSpan:last-of-type': {
  ```
In case `children` also contains a tooltip, it will not be triggered! 
```